components/emoji.js

```diff
@@ -0,0 +1,30 @@
+import { memo, useState, useEffect } from 'react'
+import { LazyLoadImage } from 'react-lazy-load-image-component'
+
+const Emoji = ({ name }) => {
+    let [image, setImage] = useState()
+    useEffect(() => {
+        try {
+            fetch("https://emoji-ranker.now.sh/custompng")
```

Comment:
This approach means that every custom emoji will make a network request to the same endpoint. For mentions, that works because 1) they’re memoized 2) we have to hit a different endpoint for each username, but here, it’s pulling the same list. The easiest way to fix that I think would be setting a cache header on the API response, but if that doesn’t work, perhaps moving it to static props makes sense?